pom.xml

```diff
@@ -349,6 +355,14 @@
           <excludeFilterFile>spotbugs-exclude.xml</excludeFilterFile>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
We shouldn't have to explicitly set this in the compiler config.  The "maven.compiler.release"property will set this.  Also, do we have to bump this project from Java 8 to Java 11?  If so, this will trigger a major version change.

Reply:
I went ahead and bumped the version to 3.0.0-SNAPSHOT and set <maven.compiler.release>11</maven.compiler.release>.  A previous change made this library incompatible with Java 8.